packages/block-editor/src/components/block-switcher/block-styles-menu.js

```diff
@@ -4,28 +4,49 @@
 import { __ } from '@wordpress/i18n';
 import { MenuGroup } from '@wordpress/components';
 import { useState } from '@wordpress/element';
+import { useSelect } from '@wordpress/data';
+import { cloneBlock, getBlockFromExample } from '@wordpress/blocks';
 
 /**
  * Internal dependencies
  */
 import BlockStyles from '../block-styles';
 import PreviewBlockPopover from './preview-block-popover';
 
-export default function BlockStylesMenu( { hoveredBlock, onSwitch } ) {
+export default function BlockStylesMenu( {
+	hoveredBlock: { name, clientId },
+	onSwitch,
+} ) {
 	const [ hoveredClassName, setHoveredClassName ] = useState();
+	const blockType = useSelect(
+		( select ) => select( 'core/blocks' ).getBlockType( name ),
+		[ name ]
+	);
+
 	return (
 		<MenuGroup
 			label={ __( 'Styles' ) }
 			className="block-editor-block-switcher__styles__menugroup"
 		>
 			{ hoveredClassName && (
 				<PreviewBlockPopover
-					hoveredBlock={ hoveredBlock }
-					hoveredClassName={ hoveredClassName }
+					blocks={
```

Comment:
Block styles can be applied only when a single block is selected. It's only changing the CSS class name applied so it feels like `blockType.example` shouldn't be needed here. However, it looks like some sort of design decision from the past. Maybe to account for the case when a block has not enough or too much content 🤷🏻‍♂️

Reply:
Yeah, I didn't look into it deeper for now, but it looks like it's been like this for the reason you mentioned (existing content..)